tools/infrastructure/api_compare.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python
+
+import json
+import sys
+import argparse
+import xml.etree.ElementTree as ET
+from collections import namedtuple
+reload(sys)
+
+# Command line arguments parsing
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-f1", "--mobile_api_file",
+    type=str, default="MOBILE_API.xml",
+    help='Enter mobile api file name! ("MOBILE_API.xml" by default)'
+)
+
+parser.add_argument(
+    "-f2", "--hmi_api_file",
```

Comment:
@LevchenkoS I don't think so. As in Python usually for input files usef flag `-i`, for output files : `-o`, for just files: '-f'. Therefore if several files needed I used '-f1, -f2', what is described in help(option `-h`)